src/libraries/Microsoft.Extensions.HostFactoryResolver/tests/HostFactoryResolverTests.cs

```diff
@@ -6,14 +6,13 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Reflection;
 using System.Threading;
-using Microsoft.DotNet.RemoteExecutor;
 using Xunit;
 
 namespace Microsoft.Extensions.Hosting.Tests
 {
     public class HostFactoryResolverTests
     {
-        public static bool RequirementsMet => RemoteExecutor.IsSupported && PlatformDetection.IsThreadingSupported;
+        public static bool RequirementsMet => PlatformDetection.IsThreadingSupported;
```

Comment:
would it make sense to now just use `PlatformDetection.IsThreadingSupported` since `RequirementsMet` became simplified?

Reply:
Done